### PR TITLE
Restructure Cassandra event

### PR DESCRIPTION
**Pull request by medcl**

### CODE REVIEW

File: packetbeat/etc/fields.yml [hunk 1]
```diff
@@ -836,26 +836,311 @@
...
+                      type: keyword
+                      description: One string for each argument type (as CQL type).
+                - name: prepared
+                  type: group
+                  description: The result to a PREPARE message.
+                  fields:
+                    - name: prepared_id
+                      type: keyword
+                      description: Representing the prepared query ID.
+                    - name: req_meta
```
Comment (monicasarbu) [inline packetbeat/etc/fields.yml]: why not moving `req_meta` to `cassandra.request.meta` and `prepared_id` to `cassandra.request.prepared_id`?

File: packetbeat/etc/fields.yml [hunk 2]
```diff
@@ -836,26 +836,311 @@
...
+                        - name: col_count
+                          type: integer
+                          description: Representing the number of columns selected by the query that produced this result.
+                        - name: pkey_columns
+                          type: integer
+                          description: Representing the PK columns index and counts.
+                        - name: paging_state
+                          type: keyword
+                          description: The paging_state is a bytes value that should be used in QUERY/EXECUTE to continue paging and retrieve the remainder of the result for this query.
+                    - name: resp_meta
```
Comment (monicasarbu) [inline packetbeat/etc/fields.yml]: similar `resp_meta` to `cassandra.response.meta`

File: packetbeat/etc/fields.yml (hunk 1 above)
Comment (medcl) [inline packetbeat/etc/fields.yml]: `prepared_id ` and `req_meta` are belong to `response`, not `request`

Comment (monicasarbu) [inline packetbeat/etc/fields.yml]: ah, sorry. I just realized they are part of result. Nevermind!

File: packetbeat/etc/fields.yml (hunk 2 above)
Comment (monicasarbu) [inline packetbeat/etc/fields.yml]: it's ok to be part of `result`.

Comment (medcl) [inline packetbeat/etc/fields.yml]: `resp_meta` only avaiable when `result_type` is `prepared`, and there is a paired `req_meta` with `resp_meta`

**Comment by monicasarbu** [comment]
LGTM. I tested and it works fine. Thank you @medcl Waiting for green.

**Pull request merged**